symbology/simple-line-symbol/src/main/java/com/esri/samples/simple_line_symbol/SimpleLineSymbolSample.java

```diff
@@ -74,17 +79,18 @@ public void start(Stage stage) {
 
       createSymbolFunctionality(controlsVBox);
 
-      final ArcGISMap map = new ArcGISMap(Basemap.createImagery());
-
-      // set initial map view point
-      Point point = new Point(-226773, 6550477, SpatialReferences.getWebMercator());
-      Viewpoint viewpoint = new Viewpoint(point, 7200); // point, scale
-      map.setInitialViewpoint(viewpoint);
+      // create a map with the imagery basemap style
+      final ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_IMAGERY_STANDARD);
 
-      // create a view for this ArcGISMap and set ArcGISMap to it
+      // create a map view and set its map
       mapView = new MapView();
       mapView.setMap(map);
 
+      // set a viewpoint on the map view
+      Point point = new Point(-226773, 6550477, SpatialReferences.getWebMercator());
+      Viewpoint viewpoint = new Viewpoint(point, 7200); // point, scale
```

Comment:
Same suggestion as above, at the very least let's ditch the comment.

Reply:
Removed `Point` and just set `Viewpoint`